### DISCUSSION TITLE
Phishing Pages and Trademark Infringement

### DISCUSSION THREAD

**Discussion by selormofmars** [vercel/community#6552] [Help] [answered]
### Summary

Your platform is being used to host a lot of phishing sites but your abuse and support teams have been non-responsive.

The following are examples of phishing pages that have been reported without actions being taken:
[REDACTED]

Get a dedicated abuse team or this promising platform will suffer

**Comment by amyegan** [comment]
Thanks for reporting these, @selormofmars. I will share this with the team. 

It sounds like you already submitted reports via email as well. If you let me know the case numbers, I can check on them for you. 🙇

**Reply by selormofmars** [reply]
thread::T5CQa4zRI8mAOZ-kPp7h4kE::

**Reply by selormofmars** [reply]
and thread::MxGwCCTUKCmx_1xRg_PSCUE:: 
I appreciate the feedback. Thank you

**Reply by amyegan** [reply, marked as answer]
I see that all of the sites you reported were taken down. Thanks again for helping keep others safe by sending your report to our dedicated abuse team at [abuse@vercel.com](mailto:abuse@vercel.com)

**Comment by ghost** [comment]
I'm having the same impression, I've been sending reports to the Vercel Safety team and the phishing website is still standing scamming and scamming unaware users. 

Case Number: 00225422
Thread::pB1jNVnuJ_6vVB7akrXU_UE::

The vercel.app domain is already marked as suspicious on [virustotal](https://www.virustotal.com/gui/domain/vercel.app) with bad community reports.

**Reply by amyegan** [reply]
Thanks for emailing that report. I can see it is in the correct queue awaiting specialist availability. We appreciate your patience as the team works to address a large number of reports as quickly as possible.

**Reply by amyegan** [reply]
I see that your case was solved on May 3rd. Thanks again for helping keep others safe by sending your report to [abuse@vercel.com](mailto:abuse@vercel.com)

**Comment by anastasiiasavchynpf1** [comment]
Hello 
Your platform is being used to host a lot of phishing sites but your abuse and support teams have been non-responsive.

The following are examples of phishing pages that have been reported without actions being taken:
https://pancakeswapclaims343-eta.vercel.app/
https://pnbmetlife.vercel.app/
https://1inch-defi.vercel.app/#/1/simple/swap/ETH

**Reply by amyegan** [reply]
We take these reports very seriously and our support team addresses them as quickly as possible. Tickets are generally processed by the order in which they are received. Humans are involved in site investigation to ensure all appropriate actions are taken, which means responses cannot be instantaneous.

Thank you for reporting potentially harmful sites and for your patience while the reports are investigated. 

Please remember that this community is not the appropriate place to share these reports as I and the other members of this community do not have access to the systems and tools needed to handle them. You should report new potentially harmful sites to [abuse@vercel.com](mailto:abuse@vercel.com). You can follow up on any open cases with questions or additional info via email.
